Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -10030,7 +10033,433 @@ def test_get_engine_base_url(mocker):
     mocker.patch.object(demisto, 'internalHttpRequest', return_value=mock_response)
     res = get_engine_base_url('1111')
     assert res == '11.111.111.33:443'
-    
+
+
+class TestMirrorObjectInitialization:
+    """
+    Tests for the MirrorObject dataclass initialization and __post_init__ logic.
+    """
+
+    def test_initialization_with_all_fields(self, mocker) -> None:
+        """
+        Given: Valid object_url and ticket_id.
+        When:  MirrorObject is initialized.
+        Then:  The object is created with the correct attributes, and no debug message is logged.
+        """
+        # Given
+        mock_demisto_debug = mocker.patch.object(demisto, 'debug')
+        ticket_url: Optional[str] = "http://example.com/ticket/123"
+        ticket_id: Optional[str] = "JIRA-123"
+
+        # When
+        mirror_obj = MirrorObject(object_url=ticket_url, ticket_id=ticket_id)
```

Comment:
we removed the ticket_id if im not mistaken.

Reply:
It should be object_id. Basically this is an ID we use to tie the mirrored object to an incident.